src/Common/src/System/Net/IPv6AddressHelper.Common.cs

```diff
@@ -411,6 +411,13 @@ internal static void Parse(ReadOnlySpan<char> address, Span<ushort> numbers, int
                 int toIndex = NumberOfLabels - 1;
                 int fromIndex = index - 1;
 
+                // if fromIndex and toIndex are the same mean that "zero bits" are already in the correct place
```

Comment:
Nit:
"if fromIndex and toIndex are the same mean that" =>
"If fromIndex and toIndex are the same, it means that"

Reply:
ok thank's

Reply:
done